docker/Dockerfile

```diff
@@ -0,0 +1,105 @@
+# escape=`
+# syntax=docker/dockerfile:1
+
+#This Dockerfile installs fantom, haystack-defs, and xeto before running haxall init to create hx database named "var" 
+#To create it along with the compose file, use the following command for testing:
+#docker compose -f .\docker-compose.yml run --rm -it -p 8082:8082 --entrypoint /bin/bash haxall
+
+ARG JDK_VERSION=17
+
+FROM eclipse-temurin:$JDK_VERSION AS base
+
+RUN set -e; `
+  export DEBIAN_FRONTEND=noninteractive `
+  && apt-get -q update `
+  && apt-get -q install -y git unzip curl sed `
+  && rm -rf /var/lib/apt/lists/* 
+
+WORKDIR /app
+ARG FAN_REL_VER=1.0.80
+
+#FANTOM SOURCE START
+# This is building fantom from source. We need to start with downloading the latest release of 
+# fantom (zipfile) because fantom is required to build fantom. 
+# huge help: https://gist.github.com/gvenzl/1386755861fb42db492276d3864a378c
+#set is a builtin shell command:https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_25
+RUN LATEST_ZIPURL=$(curl -s "https://api.github.com/repos/fantom-lang/fantom/releases/latest" | sed -Ene '/^[[:space:]]*"browser_download_url": *"(.+)"$/s//\1/p') `
+  && curl -fsSL "${LATEST_ZIPURL}" -o fantom.zip `
+  && unzip fantom.zip -d fantom `
+  && mv fantom/fantom-* rel `
+  && chmod +x rel/bin/* `
+  && chmod +x rel/adm/* `
+  && rm -rf fantom && rm -f fantom.zip `
+  #download linux-x86_64 SWT
+  && curl -fsSL "https://www.eclipse.org/downloads/download.php?file=/eclipse/downloads/drops4/R-4.27-202303020300/swt-4.27-gtk-linux-x86_64.zip&mirror_id=1" -o swt.zip `
```

Comment:
Fantom use or ship with swt.jar anymore. remove these lines to download , unzip and install the swt.jar